### DISCUSSION TITLE
[Question] 'Extensions Update Checker' ignorelist format

### DISCUSSION THREAD

**Discussion by TFWol** [xiaoxiaoflood/firefox-scripts#144] [General Support] [answered]
I just have a couple of quick questions about the ignore list in the **Extensions Update Checker** script.
https://github.com/xiaoxiaoflood/firefox-scripts/blob/8764048e0397ba9db174344813d674c104805560/chrome/extensionsUpdateChecker.uc.js#L10

I'm guessing I should use the file/ID names from the profile extensions folder? (Windows: `%appdata%\Mozilla\Firefox\Profiles\<profile>\extensions`)

Also, what would the proper format be?

Say I want to stop being reminded about a language pack (doesn't appear in the Extension Options Menu, but it's in the previously mentioned folder) and an extension that has an ID for a name. 

Would it be like this?
```javascript
ignoreList: [
{langpack-en-GB@firefox.mozilla.org},
{12345678-1234-1234-1234-1234567890AB}
],
```

**Comment by xiaoxiaoflood** [comment, marked as answer]
> I'm guessing I should use the file/ID names from the profile extensions folder?

Correct. A quick way to get extensions ids is by opening `about:support` and looking at `Add-ons` table.

Your error is that ids are strings and as such must be wrapped in quotes (`'` or `"`). Like this:

```js
ignoreList: [
  'uBlock0@raymondhill.net',
  '{3c078156-979c-498b-8990-85f7987dd929}'//Sidebery
], 
```

**Reply by TFWol** [reply]
Got it, thanks for the clarification.
